### PR TITLE
Add basic documentation

### PR DESCRIPTION
**Pull request by sandersky**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG

* **Added** some very basic introduction and model documentation.
* **Fixed** a bug when working with a view in v1 schema.

### CODE REVIEW

**Comment by cstolli** [comment]
🌮

**Comment by cstolli** [comment]
👍

**Comment by cstolli** [comment]
👍

File: tests/dummy/app/pods/tutorial/steps/intro.js [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
+
+## Enough talk lets install this "thing".
+
+\`\`\`bash
+ember install ember-frost-bunsen
+\`\`\`
+
+## Okay I have this voodoo installed now how do I leverage it's magic?
+
+The short answer is you consume one of it's three components:
```
Comment (psbanka) [inline tests/dummy/app/pods/tutorial/steps/intro.js]: "you consume one of *its* three components"

Comment (cstolli) [inline tests/dummy/app/pods/tutorial/steps/intro.js]: haha! man I correct myself in slack on this one all the time...

Comment (sandersky) [inline tests/dummy/app/pods/tutorial/steps/intro.js]: JavaScript is my primary language, English comes second.

**Pull request merged**